internal/controller/dnsrecord_controller.go

```diff
@@ -174,6 +176,7 @@ func (r *DNSRecordReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	var reason, message string
 	err = dnsRecord.Validate()
 	if err != nil {
+		logger.V(1).Error(err, "Failed to validate DNSRecord")
```

Comment:
I don't think the level makes any difference on Error logs i.e. `logger.Error` does the same

Reply:
yes, you are correct